depthai_sdk/requirements.txt

```diff
@@ -8,4 +8,6 @@ depthai==2.21.2
 PyTurboJPEG==1.6.4
 marshmallow==3.17.0
 xmltodict
-sentry-sdk==1.21.0
\ No newline at end of file
+sentry-sdk==1.21.0
+depthai-pipeline-graph==0.0.5
+PySide2
```

Comment:
@zrezke shouldn't PySide2 be inside the depthai-pipeine-graph dependency?

Reply:
Maybe, but based on depthai-pipeline-graph's readme it looks like it was a deliberate choice not to have it as a dependency, so that any existing Qt bindings are used instead of installing new ones.